### PR TITLE
Add an xref to e-commerce docs

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2678](https://openedx.atlassian.net/browse/DOC-2678)

Adds an xref to the Dogwood version of the new Adding E-Commerce section.

### Date Needed

As soon as feasible.

### Reviewers
- [x] Subject matter expert: @srpearce 
- [x] Subject matter expert: @rlucioni 
- [ ] Doc team review (sanity check/copy edit/dev edit): 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: @nedbat 

### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### Post-review
- [ ] Add description to release notes task as a comment
- [ ] Squash commits

File: en_us/open_edx_release_notes/source/dogwood.rst [hunk 1]
```diff
@@ -181,14 +181,17 @@ the *Building and Running an Open edX Course* guide.
...
 
 =======================
-Otto Ecommerce Service
+E-commerce Service
 =======================
 
-Open edX installations can now add an ecommerce service. `Otto`_ is an extended
-Django application that manages and handles orders for product catalogs.
+Open edX installations can now add an e-commerce service. The edX
+`e-commerce service`_ is an extended Django application that manages
```
Comment (rlucioni) [inline en_us/open_edx_release_notes/source/dogwood.rst]: > The edX `e-commerce service`_ is a Django application that manages and handles orders for product catalogs.

**Comment by rlucioni** [comment]
@lamagnifica one minor comment, otherwise :+1: .

File: en_us/open_edx_release_notes/source/dogwood.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/open_edx_release_notes/source/dogwood.rst]: done

**Comment by srpearce** [comment]
:+1:

**Comment by lamagnifica** [comment]
GitHub sucks me dry once again. Closing and opening a new PR with identical info.

**Pull request closed without merging**